hknweb/academics/models/course_surveys.py

```diff
@@ -1,17 +1,21 @@
 from django.db import models
 from django.core.validators import MinValueValidator
+from django.db.models import Case, When, Value
 
 from .base_models import AcademicEntity
+from .icsr import _chrono
 
 
 class Question(AcademicEntity):
-    pass
+    recent_semester = models.ForeignKey("Semester", on_delete=models.PROTECT, blank=True, null=True)

```

Comment:
Add these to the corresponding Serializer e.g. https://github.com/compserv/hknweb/blob/8707226e264c87e871a050c73134701dae244fa5/hknweb/academics/serializers/logistics_serializers.py#L9